docker/nginx/nginx.conf

```diff
@@ -1,7 +1,12 @@
 server {
     client_max_body_size 64M;
+    listen 443 ssl;
     listen 80;
+    server_name _;
+    ssl_certificate /etc/ssl/certs/nginx-selfsigned.crt;
```

Comment:
Don't hardcode this as self-signed here. May be use some generic names like `cert.crt` , cert-key.key so that it would be easy to volume mount an actual certifcate and document accordingly.

Reply:
will make these changes on bahmni-master